packages/asset-uploader/src/upload.ts

```diff
@@ -32,25 +32,31 @@ export const createUploadName = async (
       "You don't have access to create this project assets"
     );
   }
+  const { userId } = context.authorization;
+  if (userId === undefined) {
+    throw new Error("The user must be authenticated to create a project");
```

Comment:
Wrong error. Probably we should not ask to auth to upolad file. If user has edit permission thorugh a token why not?
mb better to write initial ProjectId here

Reply:
Added uploaderProjectId instead